### PR TITLE
Add orchestration to service config

### PR DESCRIPTION
**Pull request by dragorosson**

### CODE REVIEW

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

File: otter/constants.py [hunk 1]
```diff
@@ -36,6 +37,10 @@ def get_service_configs(config):
             'name': config['cloudLoadBalancers'],
             'region': config['region'],
         },
+        ServiceType.CLOUD_ORCHESTRATION: {
```
Comment (manishtomar) [inline otter/constants.py]: Can this service be optional just like metrics and cloudfeeds is? It will be nice to run otter without mandating this service.

**Comment by manishtomar** [comment]
Is it possible to only have `otter/constants.py` and `otter/test/test_constants.py` in this PR since that is simple cohesive change to review? :)

File: otter/constants.py (hunk 1 above)
Comment (radix) [inline otter/constants.py]: I actually suggested making it mandatory to reflect the fact that cloud servers is also mandatory here -- in order to facilitate normal otter API operation we will need the orchestration service running just as much as we will need cloud servers.

**Comment by radix** [comment]
@manishtomar I think this PR is showing more than it should because it's based on drago's renaming branch. once that PR is merged into master and this branch is updated it should be smaller.

**Comment by radix** [comment]
jenkins please run integration tests

**Comment by manishtomar** [comment]
jenkins please run integration tests

**Comment by manishtomar** [comment]
LGTM! Merging :)

**Pull request merged**